src/helpers/getClasses.ts

```diff
@@ -52,12 +52,19 @@ export const getClasses = (
       );
     } else if (fileType === FileTypes.scss) {
       const filePath = getFilePath(fileName);
+      const sassOptions = rendererOptions.sass || {};
+      const includePaths = [filePath, 'node_modules'];
```

Comment:
Instead of the changes here, could we do:

```
.renderSync({
  data: css,
  includePaths: [filePath, 'node_modules', ...(sassOptions?.includePaths || [])],
  ...sassOptions,
})
```

Reply:
YOu are still going to have the same problem as before that I mentioned where the `sassOptions.includePaths` overwrites the  `[filePath, 'node_modules']` in the last spread:
```...sassOptions```.

You need to remove the `includedPaths` from the `sassOptions` before the spread.

It is the bug I mentioned in the initial message.

I have changed the code to this:

```javascript
const { includePaths, ...sassOptions } = rendererOptions.sass || {};

      transformedCss = sass
        .renderSync({
          data: css,
          includePaths: [filePath, 'node_modules', ...(includePaths || [])],
          ...sassOptions,
        })
        .css.toString();
```

Reply:
if you want to use optional chaining, then typescript would need to be updated to 3.7.x, are you ok with that?